hoomd/md/methods/methods.py

```diff
@@ -23,15 +23,7 @@ class Method(_HOOMDBaseObject):
     Note:
         Users should use the subclasses and not instantiate `Method` directly.
     """
-
-    def _detach(self):
```

Comment:
Is this now handled in the base class?

Reply:
Yes, this keeps popping back up some how, I deleted it in another branch as well.